packages/react/src/components/breadcrumb/Breadcrumb.stories.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+
+import { DropdownDirection } from '../header/components/navigationLink';
+import { Header } from '../header/Header';
+import { Breadcrumb } from './Breadcrumb';
+
+export default {
+  component: Breadcrumb,
+  title: 'Components/Breadcrumb',
+  parameters: {
+    controls: { expanded: true },
+  },
+  args: {
+    list: [
+      { title: 'Front page', path: '/' },
+      { title: 'Health and social services', path: '/path' },
+      { title: 'Senior services ', path: '/path/2ndLevelPath' },
+      { title: 'Informal care', path: '/path/2ndLevelPath/3rdLevelPath' },
+      { title: 'Care options', path: null },
+    ],
+  },
+};
+
+export const Example = (args) => <Breadcrumb {...args} />;
+
+export const ExampleInHeader = (args) => {
+  return (
+    <>
+      <style>
+        {`
+          @media only screen and (min-width: 576px) {
+            .breadcrumb-container {
+              margin-right: var(--spacing-xs);
+              margin-left: var(--spacing-xs);
```

Comment:
I took the margins out from the actual component, because 
- there are multiple variations how it should be aligned (new header, current nav)
- it's very easy to add an container to align it. Like hel.fi has done. 
- overriding is not as convenient as containers. There are probably an useful container or styles ready. 
- most use cases are unknown
- theme and useTheme adds complexity

Reply:
Good, good points. Time for counterpoints! And questions:
- current navigation will be removed in 3.0.0 so there will be only the new header
- it's easy to add a container but a user might expect for the component's contents to be aligned with the header's automatically. Also user has to look up the margins from somewhere so there is some effort required
- I don't know how else a user might use this component? Without hds header? Can you elaborate on possible use cases?
- how theme adds complexity? Does new header support margins for themes?

I'm not dead set opposed to expecting that a user has to add a container for this but I just want to make sure that it's the right call after viewing all the possibilities.

Reply:
Most use-cases are unknown. Use only with new nav? Or also with current one until new is out? I've heard some projects are longing for this component and also their use cases are unclear.

So _not_ adding default margins seemed better solution, when it is likely that many have to override them. Theme adds complexity, if most users have to override defaults.

Usually one parent container handles alignment of its children, rather than every child does it by themself. Margins have to match in desktop, mobile and maybe some other breakpoints too. And perhaps sometimes `margin: auto;` is used, if content is centered, when a `max-width` is reached.

There are no specific specs for breadcrumb, but you know the new nav better. Seems that it is better to set the margins of the breadcrumb like in the new header. Projects with other use cases have to override with theme, if needed.

Reply:
> Most use-cases are unknown. Use only with new nav? Or also with current one until new is out? I've heard some projects are longing for this component and also their use cases are unclear.

Does @hannu-kivimaki know anything about breadcrumb needs from projects?

It is handy that this would be usable in core and react but it does add some complexity on implementation.. How about if in core, you recommend the container to set the margins since there are no other header components in core, and in react it uses for example css-variables? **If** there aren't very different needs for this component in hds-react users.